### PR TITLE
Create the application.md file

### PR DESCRIPTION
**Pull request by rahmot**
I created the `application.md` file so that it can be linked to the application guideline item in the [navbar](https://github.com/bebatut/galaxy_mentor_network/pull/20)

### CODE REVIEW

File: content/application.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+id: application
+title: Galaxy Community Mentorship Application Guidelines
```
Comment (bebatut) [inline content/application.md:3]: Same title as in the navbar

**Review by bebatut — COMMENTED** [review]
Could you also add the `Vue` file please?

File: content/application.md (hunk 1 above)
Comment (rahmot) [reply content/application.md]: Done, thanks!

**Comment by bebatut** [comment]
It looks good. I merge. Thanks

**Pull request merged**